README.md

```diff
@@ -63,6 +63,15 @@ or follow [this article](https://docs.microsoft.com/en-us/azure/cosmos-db/create
     azure.documentdb.database=put-your-documentdb-databasename-here
```

Comment:
Change this since you already updated the properties file.

Reply:
This section is kept to give the user two options: modifying by hand or using env variables.

Reply:
Maven plugin and intellij can demo as before: checkout the code, modify the db conf and build.